docs/sources/reference/api/docker_remote_api.md

```diff
@@ -46,6 +46,22 @@ You can still call an old version of the API using
 
 ### What's new
 
+`GET /containers/(id)/copy`
+
+`PUT /containers/(id)/copy`
```

Comment:
Okay then! :+1: I'll make a change to make the `PUT` endpoints now `POST`, but add a version check on `POST /containers/(id)/copy` and call to the old copy implementation if version is `<1.17`. Sound good?